atlasdb-console/versions.lock

```diff
@@ -338,13 +347,13 @@
             ]
         },
         "io.dropwizard:dropwizard-jackson": {
-            "locked": "0.9.3",
+            "locked": "1.0.6",
             "transitive": [
                 "com.palantir.atlasdb:atlasdb-config"
             ]
         },
         "io.dropwizard:dropwizard-util": {
-            "locked": "0.9.3",
+            "locked": "1.0.6",
```

Comment:
The other projects currently don't depend on `timelock-server`; I do remember it is possible to override the version-locking setup we have. Happy to take a more detailed look at this